server/service/integration_core_test.go

```diff
@@ -649,7 +649,7 @@ func (s *integrationTestSuite) createHosts(t *testing.T) []*fleet.Host {
 			NodeKey:         fmt.Sprintf("%s%d", t.Name(), i),
 			UUID:            uuid.New().String(),
 			Hostname:        fmt.Sprintf("%sfoo.local%d", t.Name(), i),
-			Platform:        "linux",
+			Platform:        "linux", // TODO(mna): we should use some other Linuxes here, to test our consistent grouping of e.g. darwin, rhel under "linux"
```

Comment:
I'll tackle this (integration tests with some variations of linux platform identifiers) in another PR, I wanted to get the check out first and quickly before the freeze.